Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -274,24 +275,24 @@ void FullscreenManager::cancelFullscreen()
 
     m_pendingExitFullscreen = true;
 
-    m_document.eventLoop().queueTask(TaskSource::MediaElement, [this, &topDocument, identifier = LOGIDENTIFIER] {
-        if (!topDocument.page()) {
+    m_document.eventLoop().queueTask(TaskSource::MediaElement, [this, topDocument = WTFMove(topDocument), identifier = LOGIDENTIFIER] {
+        if (!topDocument->page()) {
             INFO_LOG(identifier, "Top document has no page.");
             return;
         }
 
         // This triggers finishExitFullscreen with ExitMode::Resize, which fully exits the document.
-        if (auto* fullscreenElement = topDocument.fullscreenManager().fullscreenElement())
+        if (auto* fullscreenElement = topDocument->fullscreenManager().fullscreenElement())
             page()->chrome().client().exitFullScreenForElement(fullscreenElement);
         else
             INFO_LOG(identifier, "Top document has no fullscreen element");
     });
 }
 
 // https://fullscreen.spec.whatwg.org/#collect-documents-to-unfullscreen
-static Vector<Ref<Document>> documentsToUnfullscreen(Document& firstDocument)
+static Vector<Ref<Document>> documentsToUnfullscreen(const Ref<Document>& firstDocument)
```

Comment:
cc @rniwa on this to see if this is the new preferred style. I liked the old code better personally.

Reply:
I heard back for Ryosuke, the old style is still preferred. Please omit this change.